.github/workflows/admin.yml

```diff
@@ -7,26 +7,32 @@ on:
       - release-*
     tags:
       - v[0-9]+.[0-9]+.[0-9]+*
+
   pull_request:
     paths-ignore:
       - "agent/**"
       - "managed/**"
       - "docs/**"
 
+  workflow_call:
+    inputs:
+      go-version:
+        description: An array of Go versions to be tested against, in a format of json string.
+        required: false
+        type: string
+        default: '["tip"]'
+
 jobs:
   test-admin:
     name: Test pmm-admin
     runs-on: ubuntu-22.04
     strategy:
       fail-fast: false
       matrix:
-        go-version: [ 1.18.x ]
-        may-fail: [ false ]
-    #        include:
-    #          - go-version: tip
-    #            may-fail: true
+        go-version: ${{ fromJson(inputs.go-version || '["1.18.x"]') }}
 
     env:
+      GO_VERSION: ${{ matrix.go-version }}
```

Comment:
In fact this env variable is not needed anymore, you can use `${{ matrix.go-version }}` directly everywhere